corehq/apps/users/dbaccessors/all_commcare_users.py

```diff
@@ -6,21 +6,26 @@
 
 def get_all_commcare_users_by_domain(domain):
     """Returns all CommCareUsers by domain regardless of their active status"""
-    from corehq.apps.users.models import CommCareUser
+    return imap(CommCareUser.wrap, iter_docs(CommCareUser.get_db(), _get_ids_by_domain(domain)))
+
+
+def get_count_of_commcare_users_in_domain(domain):
+    """Returns count of all CommCareUsers by domain regardless of their active status"""
+    return len(list(_get_ids_by_domain(domain)))
```

Comment:
This doesn't seem like an efficient way to get a count, when you could just use a reduce view. Also, you're not even interested in a count—you're interested in knowing _whether there are any_ users in a domain, so both a reduce or a limit=1 would suffice to answer the question. (Either is fine with me.) 

A really simple solution would be to add a function that works off `by_domain_doc_type_date/view` called `get_doc_count_in_domain_by_class` analogous to https://github.com/dimagi/commcare-hq/blob/master/corehq/apps/domain/dbaccessors.py#L6, and then use `get_doc_count_in_domain_by_class(domain, CommCareUser) > 0`